weather_mv/loader_pipeline/regrid.py

```diff
@@ -236,6 +239,10 @@ def apply(self, uri: str):
         if self.dry_run:
             return
 
+        if not self.force_regrid and path_exists(self.target_from(uri)):
+            logger.info(f"Skipping {uri}")
```

Comment:
nit: `.`.

Reply:
Also lets add this check for zarr as well i.e. at line [272](https://github.com/google/weather-tools/pull/428/files#diff-7ee93f0ad7c757f82a6e90ea74dc1bdd9c4d04fb1f6c6b9539cd20fe7f2964a6R272).

Reply:
Refer to my below comment. Let's move `self.force_regrid` check in skip function.

Reply:
done